Scripts/generate_release.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/env bash
```

Comment:
As the effective release manager I'll probably be modifying this file after this PR merge.
Good to have as the initial example still though

Reply:
Sounds good